ecl/eclagent/eclagent.ipp

```diff
@@ -43,6 +43,8 @@ class EclGraph;
 typedef unsigned __int64 graphid_t;
 typedef unsigned __int64 activityid_t;
 
+extern Owned<IPropertyTree> agentTopology;
```

Comment:
Would be cleaner having agentTopology as a member of eclagent, rather than exposing it as a global variable.

Reply:
I didn't want to refactor to that extent. Plus you can argue that global options belong in a global variable...

It used to be both, which was very confusing.